src/components/Search.jsx

```diff
@@ -22,11 +22,17 @@ function Search() {
         setTitles(data.titles);
       });
   };
+  console.log('hello');
   useEffect(() => getRepo(), []);
   console.log(title, ids, titles, posters, taglines);
-  const Add = (e) => {
-    e.preventDefault();
-    fetch(`/api/add/${e}`);
+  const Add = (item, $event) => {
+    console.log($event, item);
+    $event.preventDefault();
```

Comment:
is preventDefault() necessary? If it's unnecessary, do you need access to the variable $event ? This doesn't look like it's a form, and the button type is not 'submit'